core/templates/dev/head/collection_editor/collection_editor_navbar_directive.html

```diff
@@ -0,0 +1,86 @@
+<script type="text/ng-template" id="inline/collection_editor_navbar_directive">
+  <div
+    class="nav navbar-nav oppia-navbar-nav navbar-right ng-cloak"
+    style="margin-left: 20px;">
+    <!-- Editor -->
+    <li ng-class="{'active': getTabStatuses().active === 'main'}">
+      <a
+        href="#"
+        tooltip="Editor"
+        ng-click="selectMainTab()"
+        class="protractor-test-main-tab">
+        <span class="glyphicon glyphicon-pencil"></span>
+      </a>
+    </li>
+
+    <!-- Preview -->
+    {% if SHOW_COLLECTION_NAVIGATION_TAB_PREVIEW and username %}
```

Comment:
I think it's a nice concept to be able to go to the thing you're working on to see how it looks for others. It's not clear yet how preview mode will work, but it would be kind of interesting if we allowed the user to specify which explorations they completed and then see what the learner view looks like for users in that situation. Seems useful. Thoughts?